src/context/online.rs

```diff
@@ -344,6 +344,26 @@ impl AudioContext {
         self.base().clear_event_handler(EventType::SinkChange);
     }
 
+    #[allow(clippy::missing_panics_doc)]
+    pub fn run_diagnostics<F: Fn(String) + Send + 'static>(&self, callback: F) {
```

Comment:
Wondering if we should mark this method `#[doc(hidden)]` because I could imagine we are not keeping the API stable over different versions. Would be interesting to return more structured data in the future

Reply:
yup looks reasonable